src/core/client/framework/helpers/getMe.ts

```diff
@@ -3,7 +3,9 @@ import { Environment, ROOT_ID } from "relay-runtime";
 export default function getMe(environment: Environment) {
   const source = environment.getStore().getSource();
   const root = source.get(ROOT_ID)!;
-  const meKey = Object.keys(root).find(s => s.startsWith("me("))!;
+  const meKey = Object.keys(root)
```

Comment:
Nice! So I'm assuming this gets the `me` out of the Relay Store. Right? We can make a new section in our docs for our helpers

Reply:
Yes! It's only useful when handling mutations. Otherwise you'd get me using a fragment.